.github/workflows/ci.yml

```
@@ -51,3 +51,8 @@ jobs:
                   HAZELCAST_ENTERPRISE_KEY: ${{ secrets.HAZELCAST_ENTERPRISE_KEY }}
               run: |
                   npm run coverage
+            - name: Publish to Codecov
+              if: ${{matrix.os == 'ubuntu-latest'}}
```

nit: for consistency we can add spaces 
```suggestion
              if: ${{ matrix.os == 'ubuntu-latest' }}
```